content/english/supply chain analytics/4_SolverAndSolving.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Solving Your Problem"
+date: 2021-03-13T14:17:07.42-07:00
+draft: false
+weight: 5
+---
+
+To run an optimization problem in Excel, you need to enable the add-on called “Solver.” Follow the link below for details or see the image below. 
+
+![Alt text](img/Download%20Excel%20solver%20img.jpg)
```

Comment:
We don't have to do this now, but we can potentially walk through the steps as a GIF.